.github/workflows/_test_bare_metal.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         runs-on: [ macos-14, macos-13, macos-12, ubuntu-22.04, ubuntu-20.04, windows-latest, arm-4core-linux ]
-        go-version: [ '1.22', '1.21', '1.20' ]
```

Comment:
No `1.23-rc` here?

Reply:
Nope, cf. [this](https://github.com/actions/go-versions/blob/main/versions-manifest.json)